src/PythonMigrationViewExtension/GraphPythonDependencies.cs

```diff
@@ -33,43 +33,52 @@ internal GraphPythonDependencies(ViewLoadedParams viewLoadedParams)
 
         private static bool IsIronPythonPackageLoaded()
         {
-            PythonEngineSelector.Instance.GetEvaluatorInfo(PythonEngineVersion.IronPython2,
-                out string evaluatorClass, out string evaluationMethod);
-            if (evaluatorClass == PythonEngineSelector.Instance.IronPythonEvaluatorClass &&
-                evaluationMethod == PythonEngineSelector.Instance.IronPythonEvaluationMethod)
-            {
-                return true;
-            }
-            return false;
+            PythonEngineSelector.Instance.GetEvaluatorInfo(
+                PythonEngineVersion.IronPython2,
+                out string evaluatorClass,
+                out string evaluationMethod);
+
+            return evaluatorClass == PythonEngineSelector.Instance.IronPythonEvaluatorClass
+                && evaluationMethod == PythonEngineSelector.Instance.IronPythonEvaluationMethod;
         }
 
-        internal bool ContainsIronPythonDependencyInCurrentWS()
+        /// <summary>
+        /// Determines if the current workspace has any dependencies on IronPython engine.
+        /// </summary>
+        /// <returns>True if depencies are found, false otherwise.</returns>
+        internal bool CurrentWorkspaceHasIronPythonDepency()
```

Comment:
@reddyashish please review these changes.

Reply:
will revert the behaviour as part of other comment below, but please have a second sanity check @reddyashish

Reply:
Is there any reason why this class has been touched? Are these changes relevant to the work for this PR?

Reply:
I see you have reverted most changes and only done some code cleanup preserving the logic. 👍